src/app/admin/appointment/appointment-admin.component.ts

```diff
@@ -46,21 +44,15 @@ export class AppointmentAdminComponent {
    * @return {string}      Local hour in format 'HH:mm'
    */
   printHour(hour: number): string {
-    // add increment
-    const hourNew = this.calcHour(hour, this.addHours);
+    let hourNew = hour + this.increment * 100;
+    hourNew = hourNew < 0 || hourNew >= 2400 ? 0 : hourNew;
```

Comment:
Is this really correct? If `hour` is at 2300 and `increment` at 3 then it should result in 0200, shouldn't it?